container/import.bzl

```diff
@@ -140,7 +156,8 @@ def _container_import_impl(ctx):
 container_import = rule(
     attrs = dict({
         "config": attr.label(allow_files = [".json"]),
-        "layers": attr.label_list(allow_files = tar_filetype + tgz_filetype),
+        "manifest": attr.label(allow_files = [".json"], mandatory = False),
+        "layers": attr.label_list(allow_files = tar_filetype + tgz_filetype, mandatory = False),
```

Comment:
why is layers now optional. I saw the comment on the PR that states "This allows Bazel to import a Windows base image without checking in GBs of tar files" but I'm not sure I understand why having layers for windows image import is problematic (why it results in checking in GBs of tar files). Also, would it not be valid to just pass an empty list for layers?